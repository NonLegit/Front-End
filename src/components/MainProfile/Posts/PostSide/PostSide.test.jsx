import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
// import { BrowserRouter as Router } from 'react-router-dom';
// import MainProfile from '../../MainProfile';
import PostSide from './PostSide';
// import ListingContextProvider from '../../../../contexts/ListingContext';

test('test snapshot', async () => {
  const tree = renderer.create(
    // <Router>
    //   <ListingContextProvider>
    //     <MainProfile>
    <PostSide />,
    //     </MainProfile>
    //   </ListingContextProvider>
    // </Router>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
