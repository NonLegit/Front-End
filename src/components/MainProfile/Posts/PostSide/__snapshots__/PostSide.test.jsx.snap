// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot 1`] = `
<div
  className="MuiBox-root css-1f5p6w6"
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1m9p8mk-MuiSvgIcon-root"
    data-testid="ForwardOutlinedIcon"
    focusable="false"
    onClick={[Function]}
    viewBox="0 0 24 24"
  >
    <path
      d="M14 8.83 17.17 12 14 15.17V14H6v-4h8V8.83M12 4v4H4v8h8v4l8-8-8-8z"
    />
  </svg>
  <span
    className="MuiTypography-root MuiTypography-caption css-sfqsx-MuiTypography-root"
  />
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-hshnfv-MuiSvgIcon-root"
    data-testid="ForwardOutlinedIcon"
    focusable="false"
    onClick={[Function]}
    viewBox="0 0 24 24"
  >
    <path
      d="M14 8.83 17.17 12 14 15.17V14H6v-4h8V8.83M12 4v4H4v8h8v4l8-8-8-8z"
    />
  </svg>
</div>
`;
